src/artemis/parameters/plan_specific/rotation_scan_internal_params.py

```diff
@@ -26,8 +26,8 @@ class RotationScanParams(BaseModel, AbstractExperimentParameterBase):
     image_width: float = 0.1
     omega_start: float = 0.0
     phi_start: float = 0.0
-    chi_start: Optional[float] = None
-    kappa_start: Optional[float] = None
+    chi_start: float | None = None
+    kappa_start: float | None = None
```

Comment:
Could: Probably a new ticket but sort of feels like all of these are optional (apart from maybe the one you are rotating)  and if not provided you just use the current value

Reply:
Agree in principle but that's an issue with the rotation scan plan, not nexus writing